### DISCUSSION TITLE
Gatsby not installing

### DISCUSSION THREAD

**Discussion by Tridence** [gatsbyjs/gatsby#30983] [Help] [answered]
## Description

I have tried to install Gatsby on windows machine using VCS. I am currently using node version v12.16.2

### Steps to reproduce
The only step I used to reproduce is npm install -g gatsby-cli

How to Make a Minimal Reproduction: https://www.gatsbyjs.com/contributing/how-to-make-a-reproducible-test-case/

### Expected result

I should be able to see a succesfully installed message but in this case, 

### Actual result

npm WARN registry Unexpected warning for https://registry.npmjs.org/: Miscellaneous Warning EINTEGRITY: sha512-jYBTV2n1gegQ+CtGOQ0I0wFkY4oPzCPXTF9Wh81ItTcKXce2uT9vn6YWfrSubhqEwwHFfWlCk6JXcelv+iJyag== integrity checksum failed when using sha512: wanted sha512-jYBTV2n1gegQ+CtGOQ0I0wFkY4oPzCPXTF9Wh81ItTcKXce2uT9vn6YWfrSubhqEwwHFfWlCk6JXcelv+iJyag== but got sha512-30JMK/urXCtHD/SSXZNTfM3er9bJ5PoFeVecJACJWZIotmL+M223MWBxQ0v14c5aHd4LMv3NQaE5NMMLXCJJpQ==. (104521 bytes)
npm WARN registry Using stale data from https://registry.npmjs.org/ due to a request error during revalidation.
npm ERR! code EINTEGRITY
npm ERR! errno EINTEGRITY
npm ERR! Invalid response body while trying to fetch https://registry.npmjs.org/file-type: Integrity verification failed for sha512-jYBTV2n1gegQ+CtGOQ0I0wFkY4oPzCPXTF9Wh81ItTcKXce2uT9vn6YWfrSubhqEwwHFfWlCk6JXcelv+iJyag== (C:\Users\TRIDENCE\AppData\Roaming\npm-cache\_cacache\content-v2\sha512\8d\80\535769f581e810f82b46390d08d30164638a0fcc23d74c5f5687cd48b5370a5dc7b6b93f6f9fa6167eb4ae6e1a84c301c57d694293a25771e96ffa22726a)

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\TRIDENCE\AppData\Roaming\npm-cache\_logs\2021-04-21T12_48_06_497Z-debug.log

### Environment

I cant even run the Gatsby info command

**Comment by LekoArts** [comment, marked as answer]
This is an issue outside of Gatsby's control, please check Google for entries like https://github.com/npm/npm/issues/17146
Thanks!
